collections/room.js

```diff
@@ -0,0 +1,17 @@
+const mongoose = require('mongoose');
+
+const roomSchema = new mongoose.Schema({
+  name: String,
+  messages: [
```

Comment:
<img width="759" alt="screen shot 2016-01-12 at 11 14 44 am" src="https://cloud.githubusercontent.com/assets/3607637/12253831/c22ec702-b91d-11e5-9dc0-a1cc573daee6.png">

Already declared with alternate syntax. Currently don't see a need to access subdoc Schema instance, so I'll leave it like this.